ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -24,3 +25,37 @@ def avg_pool1d(
         ceil_mode=ceil_mode,
         data_format=data_format,
     )
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+def avg_pool3d(
+    x,
+    kernel_size,
+    stride=(1, 1, 1),
```

Comment:
Is there any particular reason to pick different values of `stride` and `padding` from the original paddle `avg_pool3d` function? If not, please put the default values here.